examples/Elasticsearch.Extensions.Logging.Example/HighVolumeWorkSimulation.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Security.Claims;
+using System.Security.Principal;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Elasticsearch.Extensions.Logging.Example
+{
+
+	/// <summary> Simulate work that logs in low volume with some time in between each log call </summary>
+	public class HighVolumeWorkSimulation : BackgroundService
+	{
+		private readonly ILogger<LowVolumeWorkSimulation> _logger;
+
+		public HighVolumeWorkSimulation(ILogger<LowVolumeWorkSimulation> logger) => _logger = logger;
+
+		protected override async Task ExecuteAsync(CancellationToken ctx)
+		{
+			for (var i = 0; i < 100_000; i++)
+			{
+				_logger.LogWarning($"We are logging way too much: {i}");
+				if (i % 1_00 == 0) await Task.Delay(1, ctx);
```

Comment:
```suggestion
				if (i % 100 == 0) await Task.Delay(1, ctx);
```

`1_00` is a weird way to format `100`, left over from when I wrote `1_000` 😄